src/operator/contrib/transformer.cc

```diff
@@ -841,5 +841,196 @@ MXNET_OPERATOR_REGISTER_UNARY(_contrib_div_sqrt_dim)
 .set_attr<FCompute>("FCompute<cpu>", DivSqrtDimForward_<cpu>)
 .set_attr<nnvm::FGradient>("FGradient", ElemwiseGradUseNone{"_contrib_div_sqrt_dim"});
 
+
+DMLC_REGISTER_PARAMETER(SldWinAttenParam);
+
+NNVM_REGISTER_OP(_contrib_sldwin_atten_mask_like)
```

Comment:
Why do we need two operators?

Reply:
One for computing score and the other one for computing context.

Reply:
Should we return both the score and the mask? I find that we have two operators currently.

Reply:
because we may reuse the mask for other attention methods.